scripts/download_artifacts.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+download_artifacts() {
+    if [[ -z $ARTIFACT_ID || -z $ENV ]];
```

Comment:
Nice job in creating a function around this! :)
I am not a big fan of using variables that have been created elsewhere.
You could pass them as parameters, for instance 

```
function f() {
    echo "Par 1: $1"
    echo "Par 2: $2"
}
```

and use it like this `f a b`